src/App.js

```diff
@@ -1,7 +1,15 @@
-import './App.css';
-
-function App() {
-  return <>Fried Chicken Guide Web App</>;
-}
-
-export default App;
+import './App.css';
+import Chicken from './Chicken';
+
+
+
+
+function App() {
+  return (<div>
+      <Chicken />

```

Comment:
try to use more descriptive names for components, states, etc ... this for example can be ResturantsList